data/representatives.csv

```diff
@@ -713,9 +713,9 @@ member count,person count,name,Division,State/Territory,Date of election,Type of
 714,,Julian Leeser,Berowra,NSW,2.7.2016,,,still_in_office,LIB
 715,,John McVeigh,Groom,Qld,2.7.2016,,,still_in_office,LIB
 716,,David Littleproud,Maranoa,Qld,2.7.2016,,,still_in_office,NP
-717,,Tim Hammond,Perth,WA,2.7.2016,,28.7.2018,retired,ALP
+717,,Tim Hammond,Perth,WA,2.7.2016,,10.5.2018,resigned,ALP
```

Comment:
Since this change was originally prepared, we'd already updated Tim Hammond in #50 

However, the date I'd entered there was the date of the by-election in which Gorman was elected. I'm assuming that the date here is the correct date, so I'm going to go with it.

@MicaelaMicaela Can you confirm that this sounds correct to you?